componentapiexample/src/main/java/net/gini/android/vision/component/GiniVisionActivity.java

```diff
@@ -126,11 +126,10 @@ public void onExtractionsReceived(Map<String, SpecificExtraction> extractions) {
                             analysisFragment.onDocumentAnalyzed();
                             stopScanAnimation();
                             showExtractions(getSingleDocumentAnalyzer().getGiniApiDocument(),
-                                    extractions);
+                                    extractions, document);
                         } else {
-                            LOG.debug(
-                                    "Document analyzed in the Analysis Screen, but not in the "
-                                            + "Analysis Screen anymore.");
+                            LOG.debug("Document analyzed in the Analysis Screen, but not in the "
```

Comment:
This log message does not make sense. I think that some parameter is missing, otherwise both strings should be joined.